Tests/Functional/ImportTest.php

```diff
@@ -77,6 +77,17 @@ class ImportTest extends TestCase
     ];
 
     protected $configurationToUseInTestInstance = [
+        'LOG' => [
```

Comment:
Is this supposed to be here?

Reply:
Yeah, I've clean it up during recent changes. But I got tired of adding and removing over and over again. So I'm for keeping it right now as this eases the development workflow.

Reply:
Ok.